src/SingleZodiac.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import Header from './Header';
+import './SingleZodiac.css';
+
+class SingleZodiac extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      json: {},
+    };
+  }
+
+  componentDidMount() {
+    const URL = 'https://aztro.sameerkumar.website/?sign=aries&day=today';
+    fetch(URL, {
+      method: 'POST',
+    })
+      .then((response) => response.json())
+      .then((json) => {
+        this.setState({ json });
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        <>
+          <Header />
+          <section className="sign-details">
+            Current Date: {this.state.json.current_date} <br />
```

Comment:
The data is displaying, but I do not like how it looks on the page.  I need to update styling